src/puppetlabs/comidi.clj

```diff
@@ -239,13 +239,11 @@
                (apply bidi/match-route route path (apply concat (seq req))))]
        (when handler
          (bidi-ring/request
-           (handler-fn handler)
+           handler
            (-> req
                (update-in [:params] merge route-params)
                (update-in [:route-params] merge route-params))
-           (apply dissoc match-context :handler (keys req))
-           )))))
-  ([route] (make-handler route identity)))
+           (apply dissoc match-context :handler (keys req))))))))
```

Comment:
I think you can remove the outermost parens in this block now, since there are no longer multiple arities of the fn.